code/game/turfs/open.dm

```diff
@@ -147,7 +147,7 @@
 		var/obj/buckled_obj
 		if(C.buckled)
 			buckled_obj = C.buckled
-			if(!(lube&GALOSHES_DONT_HELP)) //can't slip while buckled unless it's lube.
+			if(!(istype(C.buckled,/obj/vehicle/scooter) && lube))
```

Comment:
at some point sure, i have to redownload everything. i guess i will get around to doing it at some point